### DISCUSSION TITLE
Detect excluded packages that are not required (zombie config)

### DISCUSSION THREAD

**Discussion by simPod** [composer-unused/composer-unused#161] [Ideas]
I have found some excluded packages in my `extras.unused` section that are not required anymore.

Eg. I included `cache/psr-6-doctrine-bridge` in `extras.unused` but it is not present in `require`.

I was wondering it would be nice to also report them. Would it make sense?

**Comment by icanhazstring** [comment]
That would make total sense! Good catch. Should work like psalm or phpstan if you have an error ignored that is not reported anymore.

**Reply by simPod** [reply]
I checked what it would take to implement it. I guess loader result could also accept filters? Same as here https://github.com/composer-unused/composer-unused/blob/4f768192272337e8e1e5aaec2d8c0e35505ccd36/src/Loader/PackageLoader.php#L85

**Reply by simPod** [reply]
btw psalm is silent when there's ignored issue in baseline that is not present anymore 🙃
